web/src/scripts/webflow/helpers2.mjs

```diff
@@ -0,0 +1,5 @@
+export const wait = (milliseconds = 10000) => {
```

Comment:
[sand] Can we call this file something other than `helpers2`? Even just `wait.mjs` I think would make more sense.

Reply:
Yeah, I"ll deal with it in a friday Sand ticket.